### ISSUE TITLE
Rephrase the message about YAFC failing to solve the model

### ISSUE THREAD

**Issue by have-fun-was-taken** [have-fun-was-taken/yafc-ce#96]
As was mentioned in https://github.com/have-fun-was-taken/yafc-ce/issues/87#issuecomment-2041055402, YAFC does not update the model when it fails to solve the model. This fact can confuse the user if they assume that even though the model was not solved, then it was at least partially updated.

The example of the confusion is as follows:
1. The user specifies that they want 1 Copper Cable.
2. Satisfies the demand through any fitting recipe.
3. Sets the fixed number of buildings so they fail to satisfy the demand.
4. Observes that there is an error shown at the top of the screen, but the desired product is still colored as satisfied.

I suggest that we explain why the color is the same by rephrasing the top message. Right now it's as follows:
`YAFC tried to solve this model and failed. It then tried to find a deadlock loop, but failed again.`
I suggest to reword it to
`YAFC failed to solve the model. It also found no deadlock loops. The model was not updated.`

What do you guys think?

**Comment by veger** [comment]
`YAFC failed to solve the model and to find deadlock loops. As a result the model was not updated.`

This feels a little more concise to me (although I am not a native English speaker, so I could be easily wrong)
